ocp_utilities/infra.py

```diff
@@ -354,3 +357,32 @@ def create_icsp(icsp_name, repository_digest_mirrors):
     )
     icsp.deploy()
     return icsp
+
+
+def update_pull_secret(new_content_dict, secret_name, secret_namespace):
```

Comment:
Please add docstring and have some example how `new_content_dict` should look

Reply:
done